tests/qos/test_buffer.py

```diff
@@ -2435,7 +2436,7 @@ def _check_port_buffer_info_and_return(duthost, table, ids, port, expected_profi
         else:
             if is_mellanox_device(duthost):
                 buffer_items_to_check = buffer_items_to_check_dict["down"]
-            elif is_broadcom_device(duthost) and (asic_type in ['td2'] or speed <= '10000'):
+            elif is_broadcom_device(duthost) and (asic_type in ['td2', 'td3'] or speed <= '10000'):
```

Comment:
Why is 'td3' added here?

Reply:
After we updated `buffermgr` in PR https://github.com/Azure/sonic-swss/pull/2171, `buffermgrd` is creating lossless profile according to `pfc_enable` entry. However, we don't generate `PORT_QOS_MAP` entry for admin down ports. So we don't have lossless profile for admin down ports now. Is this expected behavior?